packages/quantic/cypress/e2e/default-1/sort/sort-expectations.ts

```diff
@@ -15,29 +20,46 @@ function sortExpectations(selector: SortSelector) {
       selector.listbox().should(display ? 'exist' : 'not.exist');
     },
 
-    containsOptions: (values: string[]) => {
-      values.forEach((value) => {
-        selector.option(value).should('exist');
+    sortOptionsEqual: (options: {value: string; label: string}[]) => {
+      options.forEach((option) => {
+        selector
+          .sortOption(option.value)
+          .should('exist')
+          .should('contain', option.label)
+          .logDetail(`${option} label should be equal to ${option.label}`);
       });
     },
 
     labelContains: (label: string) => {
       selector.label().should('contain', label);
     },
 
-    optionsEqual: (options: {value: string; label: string}[]) => {
-      options.forEach((option) => {
-        selector.option(option.value).should('contain', option.label);
-      });
-    },
-
     sortCriteriaInSearchRequest: (
       body: Record<string, unknown>,
       value: string
     ) => {
       expect(body.sortCriteria).to.equal(value);
     },
 
+    sortCriteriaInInitialSearchRequest: (
+      value: string,
+      useCase: useCaseEnum
+    ) => {
+      const interceptAliasToUse =
+        useCase === useCaseEnum.search
+          ? InterceptAliases.Search
+          : InterceptAliases.Insight;
```

Comment:
There is an existing function `getQueryAlias` that you can use that does exactly that.